data/map systems.txt

```diff
@@ -26689,6 +28359,62 @@ system Pukako
 			distance 532
 			period 463.83
 
+system Puppis
+	pos -231.22223 63.000002
+	government Syndicate
+	link Ain
+	link Botein
+	link Diphda
+	link Electra
+	link Ruchbah
+	asteroids "small rock" 13 3.2384
+	asteroids "medium rock" 9 4.1998
+	asteroids "large rock" 12 4.4528
+	asteroids "small metal" 1 2.8842
+	asteroids "medium metal" 25 4.6552
+	asteroids "large metal" 17 2.7324
+	minables aluminum 1 6.3654077
+	minables neodymium 5 4.8274569
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Merchants" 2000
	fleet "Large Core Merchants" 600
	fleet "Small Syndicate" 1800
	fleet "Large Syndicate" 3500
	fleet "Small Core Pirates" 1000
	fleet "Large Core Pirates" 2500
```
Non-pirates based on neighboring Diphda and Ruchbah, pirates higher because this is directly on the back route.